suite/tests/linux/syscall_pwait.cpp

```diff
@@ -242,7 +242,8 @@ main(int argc, char *argv[])
 
     data_t data_wrong = { &test_set, 0 };
 
-    if (syscall(SYS_pselect6, 0, NULL, NULL, NULL, NULL, &data_wrong) == 0) {
+    if (syscall(SYS_pselect6, 0, (fd_set *)NULL, (fd_set *)NULL, (fd_set *)NULL,
```

Comment:
See other comment: it seems like these casts are not necessary if using `nullptr`.

Reply:
I didn't want to change even more. 'nullptr' gets rid of the casts, so: sure. Done.